javascript.html.markdown

```diff
@@ -193,7 +193,7 @@ myArray0.join(";"); // = "32;false;js;12;56;90"
 // Get subarray of elements from index 1 (include) to 4 (exclude)
 myArray0.slice(1,4); // = [false,"js",12]
 
-// Remove 4 elements starting from index 2, and insert there strings
+// Remove 4 elements starting from index 2, and insert their strings
```

Comment:
```suggestion
// Remove 4 elements starting from index 2, and add instead the strings
```

Reply:
It doesn't look like a typo to me. It means to insert "there" the strings (details [here](https://www.w3schools.com/jsref/jsref_splice.asp)). However, I agree it is not very clear. I made a suggestion, though I believe the whole section could benefit from a rewrite.